packages/kbn-optimizer/limits.yml

```diff
@@ -1,111 +1,111 @@
 pageLoadAssetSize:
-  advancedSettings: 27596
-  alerting: 106936
-  apm: 64385
-  apmOss: 18996
-  beatsManagement: 188135
-  bfetch: 41874
-  canvas: 1066647
-  charts: 195358
-  cloud: 21076
-  console: 46091
-  core: 692106
-  crossClusterReplication: 65408
-  dashboard: 374194
-  dashboardEnhanced: 65646
-  dashboardMode: 22716
-  data: 900000
-  dataEnhanced: 50420
-  devTools: 38637
-  discover: 105145
-  discoverEnhanced: 42730
-  embeddable: 312874
-  embeddableEnhanced: 41145
-  enterpriseSearch: 35741
-  esUiShared: 326654
-  expressions: 224136
-  features: 31211
-  globalSearch: 43548
-  globalSearchBar: 62888
-  globalSearchProviders: 25554
-  graph: 31504
-  grokdebugger: 26779
-  home: 41661
-  indexLifecycleManagement: 107090
-  indexManagement: 140608
-  indexPatternManagement: 28222
-  infra: 184320
-  fleet: 415829
-  ingestPipelines: 58003
-  inputControlVis: 172675
-  inspector: 148711
-  kibanaLegacy: 107711
-  kibanaOverview: 56279
-  kibanaReact: 161921
-  kibanaUtils: 198829
-  lens: 96624
-  licenseManagement: 41817
-  licensing: 39008
-  lists: 202261
-  logstash: 53548
-  management: 46112
-  maps: 183610
-  mapsLegacy: 87859
-  mapsLegacyLicensing: 20214
-  ml: 82187
-  monitoring: 80000
-  navigation: 37269
-  newsfeed: 42228
-  observability: 89709
-  painlessLab: 179748
-  regionMap: 66098
-  remoteClusters: 51327
-  reporting: 183418
-  rollup: 97204
-  savedObjects: 108518
-  savedObjectsManagement: 101836
-  savedObjectsTagging: 59482
-  savedObjectsTaggingOss: 20590
-  searchprofiler: 67080
-  security: 189428
-  securityOss: 30806
-  securitySolution: 283440
-  share: 99061
-  snapshotRestore: 79032
-  spaces: 387915
-  telemetry: 91832
-  telemetryManagementSection: 52443
-  tileMap: 65337
-  timelion: 29920
-  transform: 41007
-  triggersActionsUi: 186732
-  uiActions: 97717
-  uiActionsEnhanced: 313011
-  upgradeAssistant: 81241
-  uptime: 40825
-  urlDrilldown: 70674
-  urlForwarding: 32579
-  usageCollection: 39762
-  visDefaultEditor: 50178
-  visTypeMarkdown: 30896
-  visTypeMetric: 42790
-  visTypeTable: 94934
-  visTypeTagcloud: 37575
-  visTypeTimelion: 68883
-  visTypeTimeseries: 155203
-  visTypeVega: 153573
-  visTypeVislib: 242838
-  visTypeXy: 113478
-  visualizations: 295025
-  visualize: 57431
-  watcher: 43598
-  runtimeFields: 41752
-  stackAlerts: 29684
-  presentationUtil: 28545
-  spacesOss: 18817
-  indexPatternFieldEditor: 90489
-  osquery: 107090
-  fileUpload: 25664
-  banners: 17946
-  mapsEms: 26072
-  cases: 102558
+  advancedSettings: 24980
```

Comment:
I really don't think we should reset all the pageLoadAssetSizes in this PR. Many teams have increased their limits to avoid constant violations (while keeping under the 100k recommended limit) and this would wipe all those out.